snuba/migrations/runner.py

```diff
@@ -36,6 +32,18 @@
 DIST_TABLE_NAME = "migrations_dist"
 
 
+def get_active_migration_groups() -> List[MigrationGroup]:
+    groups = []
+    for group in MigrationGroup:
+        if not (
+            group in OPTIONAL_GROUPS
+            and group.value in settings.SKIPPED_MIGRATION_GROUPS
+        ):
+            groups.append(group)
+
+    return groups
```

Comment:
```
return [group for group in MigrationGroup if not (group in OPTIONAL_GROUPS and group.value .....)]
```

You do not need to create a local list for this, nor adding element by element with append.

Reply:
That's much shorter, thanks.